recipe/meta.yaml

```diff
@@ -274,6 +274,7 @@ outputs:
         - nbdime
         - nbval
         - nodejs >=16
+        - openssl !=3.2.0
```

Comment:
My understanding was that this was thought to be a bug in libpq where it was using the wrong API for shared data and not an openssl bug as such. Do we expect a 3.2.1 to fix the breakage?